pkg/config/aliases_nix.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 // Aliases to setup package
-const (
```

Comment:
Why did we change from const to var?

Reply:
The underlying entries it depends on are now `var`s. This all comes from the fact golang does not support overwriting `const` during build.

Reply:
Indeed. I think having a comment here is important as someone reading the code may change from `var` to `const`

Reply:
To be clear the var that gets overwritten is here alongside a comment:
https://github.com/DataDog/datadog-agent/blob/f45ab0cbe91eda11094a55e0fe86c343ad3e1aca/pkg/config/setup/config_nix.go#L12-L17

This file and a few others still require changing `const` to `var` even if they are not overwritten at build because they now depend on the underlying injected `var`. For those I think it's not necessary to add a comment because compilation will immediately fail if someone tries to make the change.

LMK if that sounds good